koku/currency/__init__.py

```diff
@@ -1 +0,0 @@
-# noqa
```

Comment:
praise: I like that this `currency` folder was removed!

Reply:
thx...